### PR TITLE
Increase total retries

### PR DESCRIPTION
**Pull request by mkosiarc**
CLOUDBLD-4333

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [x] Changes to metadata also update the documentation for the metadata
- [x] Pull request has a link to an osbs-docs PR for user documentation updates
- [x] New feature can be disabled from a configuration file

### CODE REVIEW

File: atomic_reactor/utils/retries.py [hunk 1]
```diff
@@ -49,15 +48,13 @@ def hook_log_error_response_content(response, *args, **kwargs):
...
-                                  delay=HTTP_BACKOFF_FACTOR, method_whitelist=None,
-                                  raise_on_status=True):
+                                  times=HTTP_MAX_RETRIES, delay=HTTP_BACKOFF_FACTOR,
+                                  method_whitelist=None, raise_on_status=True):
     if _http_retries_disabled():
         times = 0
 
     retry = Retry(
         total=int(times),
-        connect=connect,
```
Comment (lcarva) [inline atomic_reactor/utils/retries.py:60]: > connect (int) –
>
> How many connection-related errors to retry on.
>
> These are errors raised before the request is sent to the remote server, which we assume has not triggered the server to process the request.
> 
> Set to 0 to fail on the first retry of this type.

https://urllib3.readthedocs.io/en/latest/reference/urllib3.util.html

Why are we claiming this is not needed? The default value appears to be `None`: https://github.com/urllib3/urllib3/blob/main/src/urllib3/util/retry.py#L177

Comment (rcerven) [reply atomic_reactor/utils/retries.py:60]: 'Total number of retries to allow. Takes precedence over other counts.'

**Review by rcerven — APPROVED** [review]

File: atomic_reactor/utils/retries.py (hunk 1 above)
Comment (mkosiarc) [reply atomic_reactor/utils/retries.py:60]: the total has higher priority and retries on ALL errors (the documentation is a bit vague) is says:

"Total number of retries to allow. Takes precedence over other counts."

But you don't actually need to specify separate retries attempts for different errors if you don't want to.
at the moment these 10 retries will retry on all errors.

the code is more convincing than the documentation
https://github.com/urllib3/urllib3/blob/main/src/urllib3/util/retry.py

I could provide you with the testing I have done, and from the brew tasks linked in those related jira isssues you can see that osbs did in fact retry on "connection related" error and our initial assumption as to why was wrong. Please read my comment in CLOUDBLD-4333

**Review by MartinBasti — APPROVED** [review]

**Review by lkolacek — APPROVED** [review]

File: atomic_reactor/utils/retries.py (hunk 1 above)
Comment (lcarva) [reply atomic_reactor/utils/retries.py:60]: Thank you for the explanation. I've chatted with Robert offline about this and he cleared up my confusion. Resolving this.

**Review by lcarva — APPROVED** [review]

**Pull request merged**